.gitignore

```diff
@@ -11,3 +11,4 @@ Gemfile.lock
 _site
 _gh_pages
 .jekyll-metadata
+.vscode/
```

Comment:
Please don't include this change as it is for a particular editor, but rather add it to your global git ignore instead: https://anthonygharvey.com/productivity/how_to_create_a_global_gitignore_file